@xen-orchestra/lite/src/composables/collection-sorter.composable.md

```diff
@@ -0,0 +1,40 @@
+# useCollectionSorter composable
+
+## Usage
+
+```typescript
+const { sorts, addSort, removeSort, compareFn, toggleSortDirection } =
+  useCollectionSorter(options);
+
+const sortedCollection = computed(() => myCollection.sort(compareFn));
+addSort("name", true);
+addSort("age", false);
```

Comment:
Why not having the same syntax as `initialSort` and pass `"-age"` instead?

Reply:
`initialSorts` was pretty hard to handle (an array of tuples with each second param of tuple being a boolean saying if it's asc or desc)

Additionally, sometimes `initialSorts` value could defined by the developer, right in the code. Whereas `addSort` is meant to be used programmatically, based on a input for the field name, and another input for the asc/desc choice.

I think it would add a small overhead to change this, which would probably make the code less readable.

```ts
// Something like this:
addSort(`${direction === "asc" ? "-" : ""}${fieldName}`)

// Instead of this currently
addSort(fieldName, direction === "asc")
```

Or wee could imagine to change the "direction" input value with "" or "-" directly:

```ts
addSort(`${direction}${fieldName}`)
```

This can be discussed with the team if you think it would be preferable to keep the same format between `initialSorts` and `addSort`.

Alternatively, another solution to make the code more understandable would be to use a `string` as second parameter to `addSort`:

```ts
addSort("name", "desc")
```

Reply:
Regarding not using `"-age"`, it makes sense and I agree.

Regarding boolean vs string, I don't have a strong opinion about this. A boolean would be (slightly) easier to use but maybe less readable. And a string would avoid creating a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) but I'm not sure it would make a huge difference.

Reply:
@pdonias Let's move forward on this keeping the current behavior (`boolean`).

If it's OK to you, let's merge this one.

Reply:
Sounds good!